src/content/quickstart/openshift/_index.md

```diff
@@ -26,6 +26,8 @@ please refer to the official [OpenShift documentation](https://docs.openshift.co
 
 {{% notice info %}}
 Please note that  `oc`, `aws-cli`, `terraform`, and `wget` need to be installed before the `prep_for_subm.sh` script can be run.
+Also note that the script is known to be working with [Terraform](https://releases.hashicorp.com/terraform/) version 0.12.
```

Comment:
Looks good. 
Just a thought.
Shall we say that the script uses Terraform internally and the max supported version is 0.12.12

Reply:
@sridhargaddam it says that the "maximum compatible version is 0.12.12" - isn't it clear enough?

Reply:
It is :) 
Just wanted to highlight that the script internally uses Terraform.

Reply:
Looks like @manosnoam tested with 0.12.29 and it worked. 
https://github.com/submariner-io/submariner/pull/949

Reply:
Right, on Terraform v0.12.29 it works good, I've verified again.